episodes/collaborating-team.md

```diff
@@ -0,0 +1,262 @@
+1. \(R) Merge the PR.
+1. \(R) Navigate to issue listing to confirm that the relevant issue has been
+        automatically closed.
+
+
+:::::: challenge
+
+### Practice with Pull Requests (15 minutes)
+
+In breakout groups, assign yourselves the role of Submitter (S) and Reviewer (R),
+and repeat the process described above by completing the steps below.
+If there are any issues open on the repository that can be solved very quickly,
+feel free to address those with this exercise.
+Otherwise, the Submitter can open a Pull Request to make simple changes:
+typo fixes, or some completely arbitrary changes to a file.
+If the changes made for the exercise are not improvements to the lesson,
+the Reviewer should follow all the steps but close the PR at the end instead of merging it.
+
+1. \(S) Edit a file and commit the changes to a new branch.
+2. \(S) Open a new Pull Request from this new branch to `main`.
+        If your PR will fix an open issue on your repository,
+        include "Fixes #X" at the start of the PR description,
+        where _X_ is the issue number.
+3. \(S) Request a review from the Reviewer.
+4. \(R) Review the PR, making comments and suggesting changes as appropriate.
+        When you have finished, approve the PR or request changes, based on your review.
+5. \(S) Respond to any comments from the Reviewer and/or, if any changes were requested,
+        make those and re-request a review when you have finished.
+6. \(R) When all changes have been made to your satisfaction,
+        approve the PR, thank the Submitter for their contribution,
+        and merge the PR
+        (Or close it if the changes were made for practice but should not
+        be included in the actual lesson.)
+
+::::::::::::::::
+
+
+:::::: callout
+
+## Reviewing is a Skill
+
+Making effective review comments in a PR is something you will get better at with practice - here is some general guidance to help with reviewing a PR from a collaborator:
+
+- Good reviews have empathy - consider the contributor's feelings, identify positives and try not to be judgemental.
+- Aim for improved/better rather than perfect.
+- Make sure your review comments are specific and actionable.
+- Make it clear in the comment what you want changed as part of a PR and, ideally, provide a concrete suggestion.
+- Only request changes if the content is wrong or hard to understand (remember, different does not always mean better).
+- If a conversation on a PR hasn’t been resolved by one or two back-and-forth exchanges, consider scheduling a meeting to discuss further.
+
+The Turing Way has more on [reviewing contributions](https://the-turing-way.netlify.app/collaboration/maintain-review/maintain-review-review),
+and a lot of the advice in Alex Hill's blog post,
+[_The Art of Giving and Receiving Code Reviews (Gracefully)_](https://www.alexandra-hill.com/2018/06/25/the-art-of-giving-and-receiving-code-reviews/)
+also applies to reviewing prose.
+
+::::::::::::::::
+
+
```

Comment:
If you need more help or practice with GitHub features, you can attend a GitHub skill-up session for Carpentries Maintainers and lesson developers which uses the [following curriculum](https://carpentries.github.io/github-skill-up-maintainers/). These sessions will be periodically announced on [The Carpentries calendar of events](https://carpentries.org/community/#community-events), [The Carpentries Slack]((https://carpentries.slack.com/)) and mailing lists.(https://carpentries.github.io/github-skill-up-maintainers/).